wicket/src/state/mod.rs

```diff
@@ -0,0 +1,42 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This code was just moved out of `wizard.rs` and `StatusBar` was renamed to `ServiceStatus` since it tracks state.